### ISSUE TITLE
Mass Obsoletion: focus on branch 'connective tissue disorder' (MONDO:0003900)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6742]
branch : 'connective tissue disorder' (MONDO:0003900)

obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0017955 | granulomatous autoinflammatory syndrome
http://purl.obolibrary.org/obo/MONDO_0017954 | pyogenic autoinflammatory syndrome
http://purl.obolibrary.org/obo/MONDO_0018782 | type 1 interferonopathy

**Comment by nicolevasilevsky** [comment]
[spreadsheet is here](https://docs.google.com/spreadsheets/d/1atKm8ZHHnXbmpPCYpA8TohgWlzUi8YNKlL0B1tItEDE/edit#gid=1280968668)

**Comment by LCCarmody** [comment]
ready for review. @sabrinatoro @nicolevasilevsky

**Comment by nicolevasilevsky** [comment]
@sabrinatoro I don't see this on our [tracking spreadsheet](https://docs.google.com/spreadsheets/d/1xTfYCOCGKx_svNT8PirgGDiCINV7IDqzhBk76DNe150/edit#gid=0)

**Comment by nicolevasilevsky** [comment]
Thanks @LCCarmody!

**Comment by sabrinatoro** [comment]
@LCCarmody, could you please also review the "leaves the branch" terms? 
Thank you!

**Comment by LCCarmody** [comment]
Done with these, @sabrinatoro @nicolevasilevsky I feel kinda weird that the only 2 can just leave the branch, but that really seems the case.

**Comment by sabrinatoro** [comment]
I have reviewed the spreadsheet and agree with the assessment.
This is ready to merge.

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1VOiPTk03NeF4nGRZ198jQH978UCxNAvb/edit#gid=1564939562

TO DO:
- [ ] add new comments ("ROBOT Reviewer-Curator Agree - Leave" Tab)
- [ ] remove parents that are in the connective tissue disease branch ("Reviewer-Leave Curator-Stay" Tab)
- [ ] add comments; _!!!! comment for one term needs to be done manually_ ("ROBOT Reviewer-Leave Curator-Stay" Tab)

Once the to-do list is done, all terms reviewed by experts will have been reviewed, and issue can be closed.

**Comment by sabrinatoro** [comment]
Note from PR: 

The terms that should not be classified as "connective tissue disorder" in the spreadsheet above are classified as 'connective tissue disorders' because of the parent 'autoinflammatory syndrome'.

We should reconsider the classification of 'autoinflammatory syndrome'. It is a child of 'rheumatic disorder', which is a child of 'connective tissue disorder'. I wonder if rheumatic disorder should be a child of 'connective tissue disorder'? This comes from DO and MESH.

The [CDC definition](https://www.cdc.gov/arthritis/communications/features/rheumatic-diseases-and-pain.html#:~:text=Rheumatic%20disease%20is%20an%20umbrella,that%20mainly%20affect%20the%20joints).) of rheumatic disorder is more broad:
Rheumatic disease is an umbrella term that refers to arthritis and several other conditions that affect the joints, tendons, muscle, ligaments, bones, and muscles (arthritis refers to disorders that mainly affect the joints).

Or maybe 'autoinflammatory syndrome' should not be a child of 'rheumatic disorder'? This comes from Orphanet.

We will create a separate issue to handle this

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
